doc/source/whatsnew/v2.0.0.rst

```diff
@@ -816,6 +816,7 @@ Deprecations
 - Deprecated :meth:`DataFrame.pad` in favor of :meth:`DataFrame.ffill` (:issue:`33396`)
 - Deprecated :meth:`DataFrame.backfill` in favor of :meth:`DataFrame.bfill` (:issue:`33396`)
 - Deprecated :meth:`~pandas.io.stata.StataReader.close`. Use :class:`~pandas.io.stata.StataReader` as a context manager instead (:issue:`49228`)
+- Deprecated producing a single element when iterating over a :class:`DataFrameGroupBy` or a :class:`SeriesGroupBy` that has been grouped by a ``level`` parameter that is a list of length 1; A tuple of length one will be returned instead (:issue:`51583`)
```

Comment:
Instead of "a single element", maybe "a scalar" would be better.

Reply:
Any reference to DataFrameGroupBy / SeriesGroupBy needs to be preceded with a `.` for the link to resolve. E.g.

    :class:`.DataFrameGroupBy`

Reply:
Nit: in `; A tuple of length one`, the A should be lowercase.

Reply:
Thanks for the review! Truth be told I referenced the documentation for the `by` fix (GH42795) without questioning if there was any issues with that formatting. I'll fix!